pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataCacheTest.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
I suspect you need to add this to https://github.com/pgjdbc/pgjdbc/blob/c84e62efa5b98323562753e45fbf0d974eaca483/pgjdbc/src/test/java/org/postgresql/test/jdbc2/Jdbc2TestSuite.java